com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -463,6 +488,148 @@ public override void OnInspectorGUI()
             m_SerializedCamera.serializedObject.ApplyModifiedProperties();
         }
 
+        private void UpdateStackCemerasToOverlay()
+        {
+            int cameraCount = m_SerializedCamera.cameras.arraySize;
```

Comment:
minor: I would add  couple of methods to `UniversalRenderPipelineSerializedCamera`

```
        /// <summary>
        /// Returns a <see cref="Camera"/> at a given index
        /// </summary>
        /// <param name="index">The index to fetch the camera</param>
        /// <returns><see cref="Camera"/> if the index is valid, otherwise returns null</returns>
        public Camera this[int index]
        {
            get
            {
                if (index < 0 || index >= stackCamerasCount)
                    return null;

                // Return the camera on that index
                return m_SerializedCamera.cameras
                    .GetArrayElementAtIndex(index)
                    .objectReferenceValue as Camera;
            }
        }

        /// <summary>
        /// Returns the number of cameras on the stack
        /// </summary>
        public int stackCamerasCount => cameras.arraySize;
```

The code will be self contained, and you can use `UpdateStackCemerasToOverlay()` and on `UpdateStackCamerasOutput()`